integration/experiment/energy_efficiency/cpu_activity.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python3
+#
+#  Copyright (c) 2015 - 2022, Intel Corporation
+#  SPDX-License-Identifier: BSD-3-Clause
+#
+
+'''
+Helper functions for running cpu activity agent experiments.
+'''
+
+import argparse
+import os
+
+import geopmpy.agent
+
+from experiment import launch_util
+from experiment import common_args
+from experiment import machine
+
+def setup_run_args(parser):
+    common_args.setup_run_args(parser)
+    parser.add_argument('--cpu-frequency-efficient', dest='cpu_fe',
+                        action='store', default='nan',
+                        help='The efficient frequency of the cpu (a.k.a. Fce) for this experiment')
+    parser.add_argument('--cpu-frequency-max', dest='cpu_fmax',
```

Comment:
May be confusing vs `--max-frequency` in `common_args.py` (which is a max used in a sweep range). Maybe "--cpu-ca-max-frequency'? Maybe it doesn't matter.